lms/djangoapps/grades/tasks.py

```diff
@@ -193,6 +195,19 @@ def _has_db_updated_with_new_score_bwc_v2(
     return reported_modified_time >= expected_modified_time
 
 
+def _log_db_not_updated(kwargs):
```

Comment:
Nit: don't necessarily need a separate helper for this anymore since it's only called once now.  However, if you log the `reported_modified_time` (as suggested above), you may want to keep this helper.